jcl/src/java.base/share/classes/java/lang/invoke/MethodType.java

```diff
@@ -901,14 +901,21 @@ public int parameterCount(){
 		return returnType;
 	}
 	
-	/* Return the last class in the parameterType array.
+	/**
+	 * Return the last class in the parameterType array.
 	 * This is equivalent to:
 	 * 		type.parameterType(type.parameterCount() - 1)
-	 * Will throw ArrayIndexOutOfBounds on bad indexes.  This is a subclass of IndexOutOfBoundsException
-	 * so this is valid when JCKs expect certain exceptions.
+	 * @return class of final parameter, or void.class for Java 10 and up
```

Comment:
`@return class of final parameter, or void.class for Java 10 and up` -> `@return class of final parameter, or void.class if there are no parameters`